src/config.rs

```diff
@@ -549,6 +552,7 @@ impl Tunables {
             open_command: self.open_command,
             notifications: self.notifications.unwrap_or_default(),
             image_preview: self.image_preview.map(ImagePreview::values),
+            user_gutter_width: self.user_gutter_width.unwrap_or(30 as usize),
```

Comment:
`cargo clippy` suggests `30_usize` here, but I'm pretty sure you should just be able to do `30` and inference will figure it out.

Reply:
Had problems with `30` getting inferred as `u16` but I don't have that problem anymore. Maybe the LSP that was not working properly. Works as you suggested for me now.